opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -463,13 +463,17 @@ message Line {
 // Describes a function, including its human-readable name, system name,
 // source file, and starting line number in the source.
 message Function {
-  // Name of the function, in human-readable form if available.
-  int32 name_strindex = 1; // Index into string table
-  // Name of the function, as identified by the system.
-  // For instance, it can be a C++ mangled name.
-  int32 system_name_strindex = 2; // Index into string table
-  // Source file containing the function.
-  int32 filename_strindex = 3; // Index into string table
+  // Index into Profile.string_table to the name of the function, in
+  // human-readable form.
+  // Use index 0 if value is not available.
```

Comment:
Describing the semantics of how we index strings at every field is becoming repetitive IMO.

For *_strindex fields I would suggest we focus the documentation on the semantics of this string field. The strindex itself points to the fact that this is a string index, with the same meaning universally (including index 0 for empty value).

So, maybe put the documentation simply as

```
  // Function name. Empty string if not available.
  int32 name_strindex = 1;
  // Function name, as identified by the system. For instance,
  // it can be a C++ mangled name. Empty string if not available.
  int32 system_name_strindex = 2;
  // Source file containing the function. Empty string if not available.
  int32 filename_strindex = 3;
  ...
```

?

Reply:
Applied this suggestion to use `Empty string if not available`.